lib/plugins/aws/deploy/lib/cleanupS3Bucket.js

```diff
@@ -51,10 +51,6 @@ module.exports = {
   },
 
   cleanupS3Bucket() {
-    if (this.options.noDeploy) {
```

Comment:
This exact piece of code is still available in the `info` plugin and should be removed over there.